runner/src/main.rs

```diff
@@ -1094,27 +1095,28 @@ fn run_embedmd(mode: FormatMode) -> Step {
     }
 }
 
-fn run_liche() -> Step {
-    Step::Multiple {
-        name: "liche".to_string(),
-        steps: source_files()
-            .filter(is_markdown_file)
-            .map(to_string)
-            .map(|entry| Step::Single {
-                name: entry.clone(),
-                command: Cmd::new("liche", &[
-                    "--document-root=.",
-                    // We exclude the following URLs from the checks:
-                    // - https://groups.google.com/g/project-oak-discuss : not publicly accessible
-                    // - https://crates.io/crates : returns 404 (see https://github.com/raviqqe/liche/issues/39)
-                    // - https://codecov.io/gh/project-oak/oak : unstable, and often unaccessible on CI-build
-                    "--exclude=(https://groups.google.com/g/project-oak-discuss|https://crates.io/crates|https://codecov.io/gh/project-oak/oak)",
-                    &entry,
-                ]),
-            })
-            .collect(),
-    }
-}
+// TODO(#1304): uncomment, when re-run from GitHub is fixed.
```

Comment:
I would just add an ignore attribute for clippy instead of commenting out the entire function, so at least it keeps being compiled correctly, but up to you.

Reply:
Thanks. Done :)